doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -338,20 +344,20 @@
  *       <td>step-31</td>
  *       <td> Time-dependent Stokes flow driven by temperature
  *       differences in a fluid. Adaptive meshes that change between time
- *       steps.
+ *       steps. Explicit time stepping.
  *       </td></tr>
  *
  *   <tr valign="top">
  *       <td>step-32</td>
  *       <td> A massively parallel solver for time-dependent Stokes flow driven
  *       by temperature differences in a fluid. Adapting methods for real-world
- *       equations.
+ *       equations. Explicit time stepping.
```

Comment:
Thinking about it, I suggest we change this to `Implicit/explicit time stepping`. The temperature system treats diffusivity implicitly and the convection explicitly:
https://github.com/dealii/dealii/blob/4d7085ea23b89032b6af6e53ce767f25ae207953/examples/step-32/step-32.cc#L2674-L2677

And now I realize that the same is true for step-31:
https://github.com/dealii/dealii/blob/4d7085ea23b89032b6af6e53ce767f25ae207953/examples/step-31/step-31.cc#L1501-L1504

Reply:
Ok thanks, I will do!

Reply:
Thanks, I've fixed this now.